inc/namespace.php

```diff
@@ -275,6 +276,21 @@ function override_elasticpress_feature_activation( bool $is_active, array $setti
 	return $features_activated[ $feature->slug ];
 }
 
+/**
+ * Enables the required settings for slowlog queries to be captured.
+ *
+ * @param array $mapping
+ * @return array
+ */
+function enable_slowlog_thresholds( array $mapping ) : array {
+	$config = get_config()['modules']['search'];
```

Comment:
We should default this to `true`, as I don't mostly thing it's something someone would expect to have to enabled. 500ms seems like a bit of a slow threshhold though, I think many queries take this long and I don't think we'd typically consider that to be a problem. I'd do more like 2s and 5s.

Also, this is only enabling the `query` threshold, but we should also enable the `fetch` threshold. Recent performance issues with ES I've seen have been on `fetch` not `query`.

Reply:
👍 👍 Updated!

Reply:
The default should/would be added here: https://github.com/humanmade/altis-enhanced-search/blob/master/load.php#L14

Reply:
Updated!